lib/utils/embargoedAssets.js

```diff
@@ -0,0 +1,93 @@
+import rp from 'request-promise'
+import jwt from 'jsonwebtoken'
+
+const SIX_HOURS_IN_MS = 6 * 60 * 60 * 1000
```

Comment:
@leonardofreitass during a live review we settled with this value for now.
But we're also okay to lower it further, as the logic is quite smart and would fetch a new token if necessary.
6 hours is mostly a compromise between not too long for potential abuse and not too short that you can still download a huge space in one go.
WDYT?